README.md

```diff
@@ -1,6 +1,6 @@
 # Dataportal
 
-This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.30.
+This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.4.8 (with NPM v. 3.10.10 and Node v. 6.10.3).
```

Comment:
We could also store this info in the package.json:
https://docs.npmjs.com/files/package.json#engines

Reply:
I think I'm going to do this in a separate branch. I tried updating Node, NPM, and Angular-cli and ran into some errors (not sure if there  are compatibility issues with Angular-cli and the latest Node LTS)